spec/models/tag_user_spec.rb

```diff
@@ -230,6 +230,8 @@ def watching
       end
 
       it "sets notification level to the highest one if there are multiple tags" do
+        SiteSetting.watched_precedence_over_muted = true
```

Comment:
I think that this was actually a bug, a notification was created, information that 1 new topic is available was displayed, but topic was not visible on the list.